app/controllers/activities_controller.rb

```diff
@@ -6,11 +6,11 @@ class ActivitiesController < AuthenticatedController
 
   def index
     @page = params[:page].present? ? params[:page].to_i : 1
+    @users_for_select = current_project.activities.map(&:user).union(current_project.authors.enabled)
```

Comment:
Do we want to include users that have no activities?

Reply:
With the first part of this query we're also including users who have activities in the project even if they're no longer on the project. Do we  want to include these?